cli/src/revset_util.rs

```diff
@@ -213,6 +214,37 @@ pub fn parse_immutable_heads_expression(
     Ok(heads.union(&RevsetExpression::root()))
 }
 
+/// Prints warning if `trunk()` alias cannot be resolved. This alias could be
+/// generated by `jj git init`/`clone`.
+pub(super) fn warn_unresolvable_trunk(
+    ui: &Ui,
+    repo: &dyn Repo,
+    context: &RevsetParseContext,
+) -> io::Result<()> {
+    let (_, _, revset_str) = context
+        .aliases_map()
+        .get_function("trunk", 0)
+        .expect("trunk() should be defined by default");
```

Comment:
Nit: Can we not crash here? Since some people surely will try to remove or misconfigure the `trunk()` alias.

Reply:
We don't have a way of removing config (and `trunk()` is defined by default), but can it happen if you redefine `trunk() = 1` (an integer) or something like that?

Reply:
Aliases are loaded from all config layers (to get around parameter name differences.) Invalid `trunk() = 1` will never override the default.